modules/installation-aws-config-yaml.adoc

```diff
@@ -367,6 +367,9 @@ endif::vpc,restricted[]
 ====
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
 ====
+ifndef::vpc,restricted[]
+<12> You can provide either the `NLB` or `Classic` load balancer type in this parameter. The default load balancer type is Classic if no value is provided.
```

Comment:
```suggestion
<12> You can provide either the `NLB` or `Classic` load balancer type in this parameter. If you do not specify a value, the installation program defaults to `Classic`.
```
So you can refer to the specific object reference and use backtics (`Classic`) or refer to the general object reference (classic load balancer type).

See the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) for help in this area. I still have a hard time with this concept and refer to this section of the guidelines often. Many writers default to using literals when they are not sure what to do.